libfwupdplugin/fu-dpaux-device.c

```diff
@@ -131,11 +131,20 @@ fu_dpaux_device_setup(FuDevice *device, GError **error)
 
 	/* sanity check */
 	if (priv->dpcd_ieee_oui == 0x0) {
-		g_set_error_literal(error,
-				    FWUPD_ERROR,
-				    FWUPD_ERROR_NOT_SUPPORTED,
-				    "no DPCD OUI data was provided");
-		return FALSE;
+		guint16 vid = fu_udev_device_get_subsystem_vendor(FU_UDEV_DEVICE(self));
+		guint16 pid = fu_udev_device_get_subsystem_model(FU_UDEV_DEVICE(self));
+
+		fu_device_add_instance_u16(device, "VID", vid);
+		fu_device_add_instance_u16(device, "PID", pid);
+		fu_device_build_instance_id_full(device,
```

Comment:
I wonder a better place for the system wild device entry to go to instead dpaux. Suggestions?

Reply:
I don't think it belongs here -- I think it's fu-udev-device.c

Reply:
The dpaux device finds the way to this plugin over drm in the plugin.